opm/output/eclipse/summaryReader.cpp

```diff
@@ -244,7 +244,7 @@ void SummaryReader::findDeviations(std::vector<double>& absdev_vec,std::vector<d
       }
       else if((*referance_vec)[ivar]<(*checking_vec)[jvar]){
 	// Check with Linear polized arguments, jvar
-	double time_array[3];
+	double time_array[3]; // should contain { [time of occurance after referance] , [time of occurance before referance] , [time of referance] }
```

Comment:
Side note: the s/a/b notation comes from `sed` and similar tools using regular expressions. I think we cannot assume everyone is a grognard like you and me @akva2 :-)